src/App.tsx

```diff
@@ -1,24 +1,197 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, {
+  useContext,
+  useState,
+} from 'react';
+import classNames from 'classnames';
+import {
+  StateContext,
+} from './components/TodoContext';
+import { TodoList } from './components/TodoList';
+import {
+  FILTER,
+  ACTIONS,
+  USER_ID,
+} from './utils/enums';
+import {
+  addTodo,
+  deleteTodo,
+  getTodos,
+  updateTodo,
+} from './api/todos';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [query, setQuery] = useState('');
+  const { state, dispatch } = useContext(StateContext);
+  const [newTodo, setNewTodo] = useState<null | Todo>(null);
+
+  function deleteAll() {
+    state.list.forEach(todo => {
+      if (todo.completed) {
+        deleteTodo(todo.id)
+          .then(() => getTodos(USER_ID)
+            .then(res => {
+              dispatch({ type: ACTIONS.SET_LIST, payload: res });
+            }));
+      }
```

Comment:
```suggestion
        deleteTodo(todo.id)
          .then(() => getTodos(USER_ID))
          .then(res => {
            dispatch({ type: ACTIONS.SET_LIST, payload: res });
          });
      }
```

Reply:
it`s not my fault ))) 
<img width="1280" alt="Screenshot 2023-09-19 at 21 07 58" src="https://github.com/mate-academy/react_todo-app-with-api/assets/20277989/38dc17f3-3427-4403-a18a-46de46b1490f">

Reply:
<img width="744" alt="image" src="https://github.com/mate-academy/react_todo-app-with-api/assets/70220667/159cbab2-3c56-4968-8e5c-c62968124374">
pay attention to the parentheses

Reply:
fix it everywhere